FESS-GUI/demodevice.cpp

```diff
@@ -144,16 +148,27 @@ void DemoDevice::sync()
 // Interface Overedload Functions
 //--------------------------------------------------------------------
 
+bool DemoDevice::ready()
```

Comment:
the convention for boolean names is is_______. So here, isReady(). It just makes the code read a bit more like english. e.g. if(device->isReady()){...}